Formula/wasm-micro-runtime.rb

```diff
@@ -0,0 +1,39 @@
+class WasmMicroRuntime < Formula
+  desc "WebAssembly Micro Runtime (WAMR)"
+  homepage "https://github.com/bytecodealliance/wasm-micro-runtime"
+  url "https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-05-18-2022.tar.gz"
```

Comment:
Do we want an explicit `version` field here in the form of `YYYY.MM.DD` instead? As it currently stands, I think this versioning is not always increasing for newer versions (for instance `04-23-2023` would be interpreted as older than `05-18-2022`).

Reply:
You are right I wasn't thinking about this, If you think it is better to have an explicit version I'm happy to add one, otherwise the project itself doesn't have any, yet.

Reply:
I'd rather not add our own ad-hoc version scheme that didn't come from upstream.

If they publish a new version called `1.0.0`, for example, then that will be parsed as older than `2022.05.18`.

Reply:
We have projects that have switched from CalVer to SemVer in the past and we addressed it with `version_scheme`. My point was that the existing versioning scheme from upstream, `05-18-2022`, is definitely not going to work for us. 

It sounds like the alternative of flipping the format of the date is not acceptable - then is the next action item to ask upstream to adopt a different versioning scheme (this sounds like a big ask)?

Reply:
I can ask there, it sounds like a feasible idea tbh.

Reply:
Asking them to change their version scheme is a big ask, but asking if it makes sense to ship their software with the version flipped around isn't. We can do something like that.

Reply:
I have opened up an upstream issue for this: https://github.com/bytecodealliance/wasm-micro-runtime/issues/1357